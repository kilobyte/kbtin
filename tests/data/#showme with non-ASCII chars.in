#showme ąęść
#showme abcąęśćdefабцдефгхийאבגדァアィイ
#showme abc𠀧𠀀𠀁𠀂
