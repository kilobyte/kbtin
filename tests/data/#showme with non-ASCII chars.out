ąęść
abcąęśćdefабцдефгхийאבגדァアィイ
abc𠀧𠀀𠀁𠀂
